src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -236,34 +223,61 @@ function ttn_svd(
 
       push!(tempTTN[v], el)
     end
+    remove_dups!(tempTTN[v])
```

Comment:
```suggestion
    unique!(tempTTN[v])
```

Reply:
Will rename to `_sort_unique!` or similar for now